react/DisclosureStateIndicator.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FC } from 'react'
+import {
+  DisclosureStateIndicator,
+  DisclosureStateIndicatorProps,
+} from '@vtex/disclosure'
+
+const StateIndicator: FC<DisclosureStateIndicatorProps> = ({ show, hide }) => {
+  return <DisclosureStateIndicator show={show} hide={hide} />
+}
+
+StateIndicator.displayName = 'DisclosureStateIndicator'
+
+export default StateIndicator
```

Comment:
Maybe, in this case, you could just export the npm component just to reduce code. I know that we gonna have to create all that if we want to make something specific to IO, but for now I think it would be better to just:
```suggestion
import {
  DisclosureStateIndicator,
  DisclosureStateIndicatorProps,
} from '@vtex/disclosure'

export default DisclosureStateIndicator
export DisclosureStateIndicatorProps
```